src/runner.js

```diff
@@ -27,6 +28,7 @@ export default async function runPerformanceTest (username, accessKey, argv, nam
 
     const sessionId = browser.sessionId
 
+    await browser.throttleNetwork(networkCondition)
```

Comment:
So by default, we will throttle each test with `'Regular 3G'`? Shouldn't we do it only if `-throttle` is explicitly provided?

Reply:
I would argue that a minor throttling should be the default behavior as you usually don't always have the best network conditions.

Reply:
`Regular 3G` is not minor throttling :P Not sure, what's the default option in Chrome DevTools? Or when using Lighthouse? IMO no throttling should be set by default.

Reply:
It is set by default in Lighthouse and I think we should follow that approach:

![Screen Shot 2019-04-16 at 3 49 24 PM](https://user-images.githubusercontent.com/731337/56215040-49555700-605f-11e9-81e0-d13e7ba52fbb.png)

especially because of the fact that values for `firstCPUIdle`, `firstInteractive` can only be determined if throttling is active.

Reply:
Ok, that's convincing 👍 We should explain in the code why it is always set on then.

Reply:
@yfangsl so let's have `Good 3G` as default